uint8 method
---
bool result
